noise_raster/raster_processing.py

```diff
@@ -220,14 +231,22 @@ def vectorize(in_ds, out_poly, selectedTableIndex):
     # Reclassify raster
     result = processing.run('native:reclassifybytable', alg_params)
 
+    # Reprojected reclassified raster
+    out_tif = temp_dir + "reclass_3035.tif"
+
+    # Reproject reclassified raster to EPSG:3035 in GTiff format. Polygon will be generated in EPSG:3035.
+    gdal.Warp(destNameOrDestDS=out_tif, srcDSOrSrcDSTab=result['OUTPUT'],
+              options=gdal.WarpOptions(format='GTiff', srcSRS='EPSG:25832', dstSRS='EPSG:3035',
+                                       outputType=gdal.GDT_Float32))
+
     # Open reclassified raster
-    check_ds = gdal.Open(result['OUTPUT'])
+    check_ds = gdal.Open(out_tif)
 
     # Get reclassified raster band
     band1 = check_ds.GetRasterBand(1)
 
     # Vectorize reclassified raster to create polygon noise contours
-    gdal.Polygonize(srcBand=band1, maskBand=None, outLayer=dst_layer, iPixValField=0)
+    gdal.Polygonize(srcBand=band1, maskBand=band1, outLayer=dst_layer, iPixValField=0)
```

Comment:
I would see your explanation to Flaminia above to be valuable as a comment here. The code alone doesn't transport the idea.

Reply:
@florianesser OK, I will add the comments and add the tables as constants.